.github/ISSUE_TEMPLATE/ordinary_task.md

```diff
@@ -0,0 +1,19 @@
+---
+name: Ordinary Task
+about: Use this template for ordinary tasks. It is a simple task that does not require a lot of effort to complete and must be completed within a day or two. It is a task that does not contain sub issues.
+title: Ordinary Task
+labels: "Release Plan"
```

Comment:
This should not have a label.

Reply:
I've removed the label in the [commit](https://github.com/ec-intl/.github/pull/26/commits/36b706f92a96de59ba072cb90212f90e58aecb83).